es6/report-generator.js

```diff
@@ -28,7 +28,8 @@ export function generateFileReport(file, spellingInfo) {
 
     const lineNumber = String(displayBlock.lineNumber);
     const lineNumberPadding = Array(10 - lineNumber.length).join(' ');
-    const linePrefix = `${lineNumberPadding}${lineNumber} |`;
+    const errorIndex = String(error.index);
```

Comment:
No alas, it was the file char index - https://github.com/artsy/artsy.github.io/pull/234#issuecomment-228150553 - I couldn't find the line index in there